packages/core/src/interfaces/ubi/subgraphUBIBeneficiary.ts

```diff
@@ -0,0 +1,25 @@
+export interface SubgraphUBIBeneficiary {
+    id: number;
+    userAddress: string;
+    communityAddress: string;
+    claims: number;
+    claimed: number;
+    lastClaimAt: number;
+    preLastClaimAt: number;
+    since: number;
+    state: number;
+
+    // timestamps
+    updatedAt: Date;
+}
```

Comment:
If we look at loan managers design, you will see that we only need `claimed`, `since`, and `state`. So all the other fields you can remove them.

One important note although. When a beneficiary is added or removed, this registry needs to be updated, otherwise it will not be updated on the frontend, resulting in bug reports.

Reply:
Thinking about this once again, maybe the `chainSubscriber` needs an `upsert` into this table both for BeneficiaryAdded and Beneficiary removed, only to update the state.

Reply:
done!